### PR TITLE
Fix 06-deploy-edpm.yaml playbook

### PR DESCRIPTION
**Pull request by rabi**
This is currently confusing with incorrect task naming.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

File: ci_framework/playbooks/06-deploy-edpm.yml [hunk 1]
```diff
@@ -4,30 +4,39 @@
...
       ansible.builtin.import_role:
         name: edpm_prepare
 
-    - name: Perform Podified and EDPM deployment on compute nodes with virtual baremetal
-      when: cifmw_edpm_deploy_baremetal | default('false') | bool
-      ansible.builtin.import_role:
-        name: edpm_deploy_baremetal
-
 - name: Run post_ctlplane_deploy hooks
-  when: not cifmw_edpm_deploy_baremetal | default('false') | bool
```
Comment (cjeanner) [inline ci_framework/playbooks/06-deploy-edpm.yml:25]: so this is to be run on all kind of env - baremetal and pre-provisionned?

Comment (rabi) [reply ci_framework/playbooks/06-deploy-edpm.yml:25]: Why not? That's been defined for multinode jobs too[1]. It should also run validate_podified_deployment.yml like other jobs.

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/centos-9/multinode-ci.yml#L13

**Pull request merged**